app/src/main/java/fr/neamar/kiss/result/AppResult.java

```diff
@@ -47,6 +52,9 @@ public View display(final Context context, int position, View v) {
         TextView appName = (TextView) v.findViewById(R.id.item_app_name);
         appName.setText(enrichText(appPojo.displayName));
 
+        TextView tagsView = (TextView) v.findViewById(R.id.item_app_tag);
+        tagsView.setText(enrichText(appPojo.displayTags));
```

Comment:
Maybe it might be better to only display the tags if they are a match for the current query? If you don't search for something by tag, then there is no reason to display the tag list